--- ibus-anthy-1.2.4/configure.ac.orig	2011-06-21 18:25:58.000000000 +0900
+++ ibus-anthy-1.2.4/configure.ac	2011-06-30 18:10:50.000000000 +0900
@@ -122,6 +122,37 @@ AC_ARG_WITH(kasumi-icon,
            [KASUMI_ICON_FILE="$datadir/pixmaps/kasumi.png"])
 AC_SUBST(KASUMI_ICON_FILE)
 
+dnl - check hotkeys for ibus-anthy which is used for ibus bridge mode.
+dnl - Normally you don't need this so could be HOTKEYS="no"
+AC_ARG_WITH(hotkeys,
+           [AC_HELP_STRING([--with-hotkeys=HOTKEYS],
+                   [Use hotkeys for ibus bridge mode. (available value: yes/no/keys)])],
+           [HOTKEYS="$withval"],
+           [HOTKEYS="no"])
+if test x"$HOTKEYS" = x"no"; then
+    HOTKEYS="<!-- <hotkeys>Control+space,Zenkaku_Hankaku</hotkeys> -->"
+elif test x"$HOTKEYS" = x"yes"; then
+    HOTKEYS="<hotkeys>Control+space,Zenkaku_Hankaku</hotkeys>"
+elif test x"$HOTKEYS" = x; then
+    HOTKEYS="<hotkeys>Control+space,Zenkaku_Hankaku</hotkeys>"
+else
+    HOTKEYS="<hotkeys>${HOTKEYS}</hotkeys>"
+fi
+AC_SUBST(HOTKEYS)
+
+SYMBOL_TEST="import ibus
+engine = ibus.EngineDesc('test')
+exit(not hasattr(engine, 'symbol'))
+"
+$PYTHON -c "$SYMBOL_TEST"
+if test $? -eq 0; then
+    SYMBOL="<symbol>\xe3\x81\x82</symbol>"
+else
+    SYMBOL="<!-- <symbol>\xe3\x81\x82</symbol> -->"
+fi
+SYMBOL=`printf "$SYMBOL"`
+AC_SUBST(SYMBOL)
+
 dnl - define GETTEXT_* variables
 GETTEXT_PACKAGE=ibus-anthy
 AC_SUBST(GETTEXT_PACKAGE)
@@ -146,3 +177,15 @@ m4/Makefile
 ])
 
 AC_OUTPUT
+echo "
+                IBus-Anthy version ibus_version
+
+python-config:           $PYTHON_CONFIG
+PYTHON_CFLAGS            $PYTHON_CFLAGS
+PYTHON_LIBS              $PYTHON_LIBS
+ANTHY_ZIPCODE_FILE       $ANTHY_ZIPCODE_FILE
+KASUMI_EXEC_FILE         $KASUMI_EXEC_FILE
+KASUMI_ICON_FILE         $KASUMI_ICON_FILE
+HOTKEYS                  $HOTKEYS
+SYMBOL                   $SYMBOL
+"
--- ibus-anthy-1.2.4/engine/anthy.xml.in.in.orig	2011-06-21 18:24:39.000000000 +0900
+++ ibus-anthy-1.2.4/engine/anthy.xml.in.in	2011-06-30 17:37:45.000000000 +0900
@@ -22,6 +22,8 @@
 			<longname>Anthy</longname>
 			<description>Anthy Input Method</description>
 			<rank>99</rank>
+			@HOTKEYS@
+			@SYMBOL@
 		</engine>
 	</engines>
 </component>
